src/collection-facets/facets-template.ts

```diff
@@ -215,20 +212,28 @@ export class FacetsTemplate extends LitElement {
       }
       .facet-checkbox {
         margin: 0 5px 0 0;
-        display: inline-block;
-        align-items: baseline;
+        display: flex;
+        align-items: center;
+      }
+      .facet-checkbox input:first-child {
+        margin-right: 5px;
+      }
+      .facet-checkbox input {
+        height: 15px;
+        width: 15px;
```

Comment:
safari has a distinct input style that made the input button shorter than eye icon.  this helps standardize it.

Reply:
Nice! 👍

Q: Should we be aiming to use this standardized approach in other places that we have checkboxes, for consistency? Or is it strictly for this use case?

Reply:
great question, laton! i think facet group is unique. we have been working to standardize our design as we work on new features, so i think we should revisit soon as a team if that's what we actually need across the site. 👍